auth-api/src/auth_api/services/affiliation.py

```diff
@@ -558,15 +561,16 @@ def _combine_nrs(name_requests, businesses, drafts):
                     del name_requests[nr_num]
                 else:
                     # If not in name_requests then it's a stale draft.
-                    drafts.remove(business)
+                    if remove_stale_drafts:
+                        drafts.remove(business)
```

Comment:
Needed to do this, otherwise our pagination counts would be off. 

We could filter in the entity_mapping DAO level, but I think it's best to leave it like this for now. 

That way we can test this easily on one account.. EG. Create two TEMPS, same NR... file 1.. the other shouldn't file or be removed by LEAR etc